packages/tradershub/src/features/cfd/screens/Jurisdiction/JurisdictionScreen.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { FC, useEffect, useMemo } from 'react';
+import { useAvailableMT5Accounts, useMT5AccountsList } from '@deriv/api';
+import { Provider } from '@deriv/library';
+import { qtMerge, Text } from '@deriv/quill-design';
+import { THooks } from '../../../../types';
+import { useDynamicLeverageModalState } from '../../components/DynamicLeverageContext';
+import { JurisdictionCard } from './JurisdictionCard';
+import { JurisdictionTncSection } from './JurisdictionTncSection';
+
+type TJurisdictionScreenProps = {
+    isCheckBoxChecked: boolean;
+    selectedJurisdiction: THooks.AvailableMT5Accounts['shortcode'];
+    setIsCheckBoxChecked: React.Dispatch<React.SetStateAction<boolean>>;
+    setSelectedJurisdiction: React.Dispatch<React.SetStateAction<string>>;
+};
+
+const JurisdictionScreen: FC<TJurisdictionScreenProps> = ({
+    isCheckBoxChecked,
+    selectedJurisdiction,
+    setIsCheckBoxChecked,
+    setSelectedJurisdiction,
+}) => {
```

Comment:
Is there a reason to use `FC` here if not can we remove it and stick to the old ways:
```suggestion
const JurisdictionScreen = ({
    isCheckBoxChecked,
    selectedJurisdiction,
    setIsCheckBoxChecked,
    setSelectedJurisdiction,
}: TJurisdictionScreenProps) => {
```

Reply:
No reason to keep FC, I just missed it. Updated.